ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -527,13 +530,16 @@ wan_router:
                 tunnel_interface_numbers: 100-102
               - name: ZSCALER-EXIT-POLICY-2
                 tunnel_interface_numbers: 110-112
+              - name: DIRECT-EXIT-POLICY-1
 ```
 
 1. Assign the `ZSCALER-EXIT-POLICY-1` and `ZSCALER-EXIT-POLICY-2` internet-exit policies to the Ethernet3 WAN interface.
 
 #### Local exit toward the ISP
 
-!!! Warning "Not currently supported in eos_designs."
+!!! Warning "Only supported in PREVIEW"
+
+Internet-exit policy type should be set to `direct` to locally exit toward the ISP.
```

Comment:
Can we put a small snippet regarding using it? like the deifnition plus the definition under the interface?